### DISCUSSION TITLE
Strongswan 9.7.2 and issue with palo alto ipsec?

### DISCUSSION THREAD

**Discussion by EeroV** [strongswan/strongswan#1622] [General]
Hi,

I have issue to create ipsec from strongswan 9.7.2 to palo alto. It worked some months ago and now it just stopped. Packet counter changes on our side, but not on palo alto?

Configuration looks like this:

conn xxx
        authby=secret
        type=tunnel
        left=%defaultroute
        leftid=xxxxx
        leftsubnet=xxxx
        leftauth=psk
        leftfirewall=yes
        right=xxxx
        rightsubnet=xxxx
        rightauth=psk
        ike=es256-sha256-ecp384
        esp=es256-sha256-ecp384
        keyexchange=ikev2
        keyingtries=0
        ikelifetime=8h
        lifetime=1h
        auto=route  

Yes, planning to upgrade but looks like configuration syntax has changed a lot. So, I need probably do lot of rewrite in our terraform code..

System is running latest amazon linux 2. Yes, it works fine with other vpn solutions..

Eero

**Comment by tobiasbrunner** [comment]
> Packet counter changes on our side, but not on palo alto

Sounds like an issue on the other end. Difficult to say more without additional information (#196).
